### DISCUSSION TITLE
[ente Auth] Add Feature to sort the Tags

### DISCUSSION THREAD

**Discussion by OliDMX** [ente-io/ente#2038] [Enhancements]
Currently you can create Tags and thats great, i just want them to move around, not having them randomly sorted. 
Maybe a Tab in the Settings would be cool too, to choose if you want the Tags sorted alphabetically or manually. 
But im fine already with just sorting the tags manually, its just chaos now ;) 
Thanks and i appreciate your work guys, really! Your App got so far.

**Comment by fronglemog** [comment]
This was my first wish as soon as I tried out the Tags feature. 

Tags are useful, but if finding the right tag takes more than a small fraction of the time it takes to just find the item normally, I don't find it useful.

**Comment by TSHsoft** [comment]
If Tag can be sorted by drag and drop, then it is more convenient.

**Comment by Kroooza** [comment]
Came here after searching how. Please add this.
